packages/sdk/source/services/address.contract.ts

```diff
@@ -8,11 +8,7 @@ export interface AddressDataTransferObject {
 
 export interface AddressService {
 	fromMnemonic(mnemonic: string, options?: IdentityOptions): Promise<AddressDataTransferObject>;
-	fromMultiSignature(
-		min: number,
-		publicKeys: string[],
-		options?: IdentityOptions,
-	): Promise<AddressDataTransferObject>;
+	fromMultiSignature(input: MultisignatureAddressInput): Promise<AddressDataTransferObject>;
```

Comment:
The `options` argument should stay separate from any kind of inputs as it was.

Reply:
@faustbrian fixed.